architecture/wm_azure/001-sequence.puml

```diff
@@ -0,0 +1,42 @@
+' Copyright (C) 2015-2021, Wazuh Inc.
+' Created by Wazuh, Inc. <info@wazuh.com>.
+' This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+@startuml wm_azure
+    main -> wmodules : wm_config()
+    wmodules -> config : ReadConfig()
+    config -> config : read_main_elements()
+    config -> config : Read_WModule()
+    config -> config : wm_azure_read()
+    config -> config : wm_azure_api_read()
```

Comment:
I think that using activation/desactivation feature (++,--) will allow to see the real call stack

Reply:
Good catch!